pages/common/pip-freeze.md

```diff
@@ -0,0 +1,24 @@
+# pip freeze
+
+> List installed packages in requirements format.
+> More information: <https://pip.pypa.io/en/stable/cli/pip_freeze>.
+
+- List installed packages:
+
+`pip freeze`
+
+- List installed packages and write it to the `requirements.txt` file:
+
+`pip freeze > requirements.txt`
+
+- List installed packages in a virtual environment, excluding globally installed packages:
+
+`pip freeze --local > requirements.txt`
+
+- List installed packages in the user-side:
```

Comment:
This is the wording used in the official documentation.
Do we really want to [change it](https://github.com/tldr-pages/tldr/pull/12184#discussion_r1476978763)? @kbdharun

https://pip.pypa.io/en/stable/cli/pip_freeze/#cmdoption-user
> --user
> Only output packages installed in user-site.

```suggestion
- List installed packages in the user-site:
```

Reply:
Maybe a typo in the docs, will consult upstream IG regarding this.